### DISCUSSION TITLE
Run renovate on the current git repository

### DISCUSSION THREAD

**Discussion by SomeBdyElse** [renovatebot/renovate#23803] [Suggest an Idea]
### Tell us more.

Dear renovate community!

I would like to use renovate in a single project, with the renovate config checked into the repository. My GitLab runner will start a renovate docker container, with the project checked out into the current workspace. 

```
/app$ which renovate
/usr/local/bin/renovate

/app$ ls -la
.git
.gitignore
.gitlab-ci.yml
README.md
app
app/package.json
app/package-lock.json
cms
cms/composer.json
cms/composer.lock
renovate.json
```

As far as I understand renovate will not use the current git workspace, but instead will clone the target git project(s) itself. Therefore it is recommended to create a separate git repository with the renovate config that then processes multiple target repositories. To me it would be easier to keep the renovate config in the project repository.

Would it be possible for renovate to work on the current git repository/workspace instead of cloning the git repo again?

**Comment by viceice** [comment]
check out docs for platform=local

**Comment by rarkins** [comment]
You're welcome to propose a PR if you can find an elegant way to do it. Otherwise you already know our recommendation
